packages/logger/src/config.ts

```diff
@@ -0,0 +1,64 @@
+import winston from "winston";
+
+export const LOG_LEVELS = [
+  "emerg",
+  "alert",
+  "crit",
+  "error",
+  "warning",
+  "notice",
+  "info",
+  "debug",
+] as const;
+
+export type LogLevel = typeof LOG_LEVELS[number];
+
+export type ConsoleTransport = {
+  type: "console";
+  options?: winston.transports.ConsoleTransportOptions;
+};
+
+export type FileTransport = {
+  type: "file";
+  options?: winston.transports.FileTransportOptions;
+};
+
+export type HttpTransport = {
+  type: "http";
+  options?: winston.transports.HttpTransportOptions;
+};
+
+export type StreamTransport = {
+  type: "stream";
+  options?: winston.transports.StreamTransportOptions;
+};
+
+export type Transport =
+  | ConsoleTransport
+  | FileTransport
+  | HttpTransport
+  | StreamTransport;
+
+export type LoggerConfig = {
+  /**
+   * The log level. Defaults to "info"
+   */
+  logLevel: LogLevel;
+
+  /**
+   * The NodeJS environment. Defaults to `process.env.NODE_ENV`
+   */
+  environment: string;
+
+  /**
+   * The list of transports to log to. Defaults to a single Console
+   * transport.
+   */
+  transports: Transport[];
+};
+
+export const defaultConfig: LoggerConfig = {
+  logLevel: "info",
```

Comment:
Might want to provide an environment variable like `LOG_LEVEL` to be able to configure the logger's default level from the environment.

Reply:
Might actually be a good idea to be able to provide the transport configuration as a JSON blob in an environment variable as well? This way logging could be different in different environments like staging/prod dynos without changing code.

Reply:
> Might want to provide an environment variable like LOG_LEVEL to be able to configure the logger's default level from the environment.

I don't see why this should be handled by the library when all it takes is one line to set `logLevel` to `process.env.LOG_LEVEL` on the consumer side.